tools/releases/dockerfiles/Dockerfile.kuma-dp

```diff
@@ -1,7 +1,9 @@
 FROM frolvlad/alpine-glibc:alpine-3.14_glibc-2.33
 
+ARG ENVOY_VERSION
```

Comment:
it seems like ENVOY_VERSION is not set on CI

Reply:
on it

Reply:
it should be v1.18.4 not just 1.18.4

Reply:
in `envoy.mk`

Reply:
fixed, thank you for catching this!